app/controllers/v1/submissions_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
     submission = task.submissions.new(
       framework: task.framework,
       supplier: task.supplier,
+      created_by: current_user,
```

Comment:
An update to the request specs (`spec/requests/v1/submissions_spec.rb`) to verify the user is actually recorded against the submission would be good.